test/Collision/Billard16Balls.jl

```diff
@@ -82,80 +82,80 @@ Ball = Model(
                                   collision=true))
 )
 
-Billard = Model(
+Billard = Model3D(
     world = Object3D(feature=Scene(gravityField=UniformGravityField(g=9.81, n=[0, 0, -1]), mprTolerance = 1.0e-18,
                                    enableContactDetection=true, # maximumContactDamping=1000.0,
                                    nominalLength=0.15*TableX)), # animationFile="Billard16Balls.json"
     table = Table,
     cushion = Cushion,
     ball0 = Ball,
     joint0 = FreeMotion(obj1=:world, obj2=:(ball0.ball),
-                        r=Var(init=[-0.8, -0.1, diameter/2]),
-                        v=Var(init=[3.0, 0.1, 0.0]),
-                        rot=Var(init=[pi/2, 0.0, 0.0])),
+                        r=Var(init=ModiaBase.SVector{3,Float64}(-0.8, -0.1, diameter/2)),
+                        v=Var(init=ModiaBase.SVector{3,Float64}(3.0, 0.1, 0.0)),
+                        rot=Var(init=ModiaBase.SVector{3,Float64}(pi/2, 0.0, 0.0))),
```

Comment:
Requirement for ModiaBase.SVector{3,Float64} is not nice.

Reply:
Yes I know. This is a temporary limitation:
- Once FreeMotion joints are handled internally in Modia3D (no code more as Modia equations), this problem is gone for Modia3D
- In general, the "merge" function needs to be adapted, so that arrays are automatically converted to static arrays, if the initial model used static arrays.